source/guides/first-steps/task-script.md

```diff
@@ -1,4 +1,130 @@
+    - narrate "This is a basic task script!"
+```
+
+This script will narrate the text: `This is a basic task script!` to the player attached to the script. If you use the `/ex` command to run this script - specifically, by running `/ex run example_task` in-game, *you* will be the attached player.
+
+### Building Your First Task Script in VS Code
+
+Previously, you've learned [how to create an environment in VS Code](/guides/first-steps/script-editor), the editor that we recommend for writing Denizen scripts.
+
+#### Creating the File
+
+To create your first task script, start by opening your scripts folder in VS Code.
+
+TODO SCREENSHOT (open scripts folder)
+
+From there, right click the scripts folder in the explorer menu and click the "New File" option.
+
+TODO SCREENSHOT (new file highlighted)
+
+Now you can begin writing your first task script!
+
+#### Writing the Script
+
+Let's start with the core of the script:
+
+```dscript_blue
+my_first_task:
+    type: task
+    script:
+    - narrate (sometext)
+```
+
+This example should look familiar - it's very similar to the example above.
+
+### Script Names
+
+The name of the script here is `my_first_task`. It's at the top of the indentation. Each top-level indentation entry is a separate script. Here is an example of two different scripts:
+
+```dscript_green
+my_first_task:
+    type: task
+    script:
+    - narrate "This is is task number one!"
+
+my_second_task:
+    type: task
+    script:
+    - narratee "This is task number two!"
+```
+
+This example demonstrates what having two different scripts in the same file looks like. `my_first_task` and `my_second_task` are two different scripts, and each is a fully self-contained script. Additionally, *script names* and *file names* are different. Both `my_first_task` and `my_second_task` could be contained within the same file named `my_first_scripts.dsc`.
+
+### Script Types
+
+Below the script name - on the second level of indentation - you'll see the `type` key.
+
+The `type` key is where you specify which `type` of Denizen script you're writing. In this case, we're writing a `task` script. You may have seen `world` scripts, `item` scripts, `inventory` scripts, and *maybe* even a `procedure` script. For now, let's focus on `task` scripts. You'll learn about the different types as you read through these guides, and you can find explanations about all of them from the meta documentation on the [language explanation page](https://one.denizenscript.com/denizen/lngs/container).
+
+Make sure you write `type: task` in your script on the second level of indentation, below the script name, like this:
+
+```dscript_blue
+my_first_task:
+    type: task
+```
+
+### Script Commands
+
+Below the `type` key, you'll see the `script` key. This is where you'll be doing most of your work in Denizen! Under the `script` key is where you write the content of the script - for most script types, the `script` key is where you tell Denizen what to do. 
```

Comment:
`Under the script key is where you write the content of the script - for most script types, the script key is where you tell Denizen what to do. `
This is still wrong, despite rewording (closer at least), per original comment
>`This is false ^
Consider for example world scripts. They do not have a script key. Only task, procedure, and command scripts use script.`

Most work in modern Denizen ends up under `events:` and most NPC-related scripting ends up under `steps:`
Also, only a few script types use `script:` , most don't.

This paragraph might be reworded to say that for a task script (and script types similar to it), `script` is "`is where you write the content of the script`" - and maybe mention that other script types have that done in different keys - either way, don't claim `script:` to represent a majority of script types or of scripting, as that's inaccurate.